client/client_app/templates/list_of_restaurants.html

```diff
@@ -12,6 +12,7 @@ <h2 class="text-center">Chose restaurant from a list:</h2>
                 <tr>
                     <th>Name</th>
                     <th>Type</th>
+                    <th>Tables Count</th>
```

Comment:
I'd rather move "Tables count" field from restaurants list to restaurant info. I do not think that this is so important info.

Reply:
done